ifrs17/Import/Importers.ipynb

```diff
@@ -862,25 +865,25 @@
                 "\n    var updatedCurrencies = toCommitYieldCurves.Select(x => x.Currency).Distinct();",
                 "\n    var dataNodesToUpdate = await workspace.Query<GroupOfContract>().Where(x => updatedCurrencies.Contains(x.ContractualCurrency)).Select(x => x.SystemName).ToArrayAsync();",
                 "\n    var workspaceToCompute = Workspace.CreateNew();",
-                "\n    workspaceToCompute.Initialize(x => x.FromSource(options.TargetDataSource));",
+                "\n    workspaceToCompute.Initialize(x => x.FromSource(options.TargetDataSource).DisableInitialization<RawVariable>().DisableInitialization<IfrsVariable>());",
                 "\n   ",
                 "\n    foreach (var args in allArgs) {",
                 "\n        await CommitPartitionAsync<PartitionByReportingNodeAndPeriod>(args, workspace, workspaceToCompute);",
                 "\n        var targetPartition = (Guid)(await options.TargetDataSource.Partition.GetKeyForInstanceAsync<PartitionByReportingNodeAndPeriod>(args));",
                 "\n        var defaultPartition = (Guid)(await options.TargetDataSource.Partition.GetKeyForInstanceAsync<PartitionByReportingNodeAndPeriod>(args with {Scenario = null}));",
+                "\n        var previousPartition = (Guid)(await options.TargetDataSource.Partition.GetKeyForInstanceAsync<PartitionByReportingNodeAndPeriod>(args with {Scenario = null, Year = args.Year-1, Month = 12}));",
+                "\n        await options.TargetDataSource.Partition.SetAsync<PartitionByReportingNodeAndPeriod>(null);",
                 "\n        if(ApplicationMessage.HasErrors()) return Activity.Finish().Merge(importLog);",
                 "\n        ",
                 "\n        // Avoid starting the computation if no best estimate cash flow has ever been imported ",
-                "\n        if(args.Scenario == null) {",
-                "\n            await options.TargetDataSource.Partition.SetAsync<PartitionByReportingNodeAndPeriod>(null);",
-                "\n            if(!(await options.TargetDataSource.Query<RawVariable>().Where(x => x.Partition == targetPartition).Take(1).ToArrayAsync()).Any()) continue;",
-                "\n        }",
+                "\n        if(!(await options.TargetDataSource.Query<RawVariable>().Where(x => x.Partition == defaultPartition).Take(1).ToArrayAsync()).Any() &&",
+                "\n           !(await options.TargetDataSource.Query<IfrsVariable>().Where(x => x.Partition == defaultPartition).Take(1).ToArrayAsync()).Any()) continue;",
+                "\n        ",
+                "\n        // Only nominals corresponding to the target data nodes are added to the workspace",
```

Comment:
rename (and comment) : instead of nominal just say rawVariables. 
(Now Nominals is a computed EconomicBasis)